index.html

```diff
@@ -1 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="nav-bar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <button type="button" class="hamburger">&#9776;</button>
+        </nav>
+    </header>
+    <main>
+        <section class="main-content">
+            <div class="content">
+                <h1>I'm Rabecca Glad to see you!</h1>
```

Comment:
- [ ] Please increase the font weight for the main heading and make sure it matches the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185749172-036c97ad-9e07-402c-8df3-a095b13eef12.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185749214-694991ff-8a56-4ba9-8993-c476f9c971f5.png)